offload/DeviceRTL/include/Synchronization.h

```diff
@@ -26,6 +26,14 @@ enum OrderingTy {
   seq_cst = __ATOMIC_SEQ_CST,
 };
 
+enum ScopeTy {
+  system = __MEMORY_SCOPE_SYSTEM,
+  device_ = __MEMORY_SCOPE_DEVICE,
```

Comment:
Is this on purpose with `_` to not conflict with something?

Reply:
Yes, was too lazy to unify it right this moment since I don't want to delete the CUDA handling yet. Maybe someone like @AlexMaclean knows the status of lowering stuff like https://godbolt.org/z/cWs77s9Eq accurately for NVPTX without NVVM.